src/passes.jl

```diff
@@ -306,6 +306,7 @@ end
 function process!(::SetupBuildDirectory, page::Page, content, block)
     dir = dirname(page.build)
     isdir(dir) || mkpath(dir)
+    addstaticfiles!(dir)
```

Comment:
> append! staticfiles to document.pages

Sorry, missed that part. Adding them as `Page`s would probably been best. Something like

```diff
diff --git a/src/types.jl b/src/types.jl
index 6533bd6..e9d5244 100644
--- a/src/types.jl
+++ b/src/types.jl
@@ -41,6 +41,13 @@ immutable Document
                 doc.pagemap[obj] = page
             end
         end
+        assetdir = normpath(joinpath(dirname(@__FILE__), "..", "assets"))
+        for asset in readdir(assetdir)
+            obj, out = paths(assetdir, asset, assetdir, build)
+            page = Page(doc, obj, out, format)
+            push!(doc.pages, page)
+            doc.pagemap[obj] = page
+        end
         doc
     end
 end
```

and store each static file in a `Lapidary/assets` dir as an actual file instead of the `Dict` in `staticfiles.jl`. That way they're easier to edit in their actual format rather than a Julia string.

(Any project-specific asset files can just be put into the user's `docs/src` dir and they will get copied over.)